server/sonar-server/src/main/java/org/sonar/server/computation/batch/BatchReportReaderImpl.java

```diff
@@ -53,6 +55,20 @@ public BatchReportReaderImpl(BatchReportDirectoryHolder batchReportDirectoryHold
   }
 
   @Override
+  public Optional<CloseableIterator<String>> readScannerLogs() {
+    File file = delegate.getFileStructure().analysisLog();
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) org.sonar.server.computation.batch.BatchReportReaderImpl must not use org.sonar.batch.protocol.output.FileStructure [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AThe_Server_stack_should_not_access_classes_of_the_Batch_stack)